Tests/test_dynamic_loading.py

```diff
@@ -82,6 +82,11 @@ def test_can_import_trigger(self):
         triggers = self.get_package_in_folder(self.trigger_dir)
         package_name = "trigger"
         for trigger in triggers:
+
+            # snowboy is only compiled for python2
```

Comment:
And so the new binary must be a part of what we deliver too. We don't want to ask the user to compile something manually.